m4_sessions/1806-inning/group9.md

```diff
@@ -1,34 +1,55 @@
 ## Group Members:
+Evan Wheeler
+Andrew Piermarini
+Krista Handel
 
 
+## Name of Session: Privacy and Security Awareness in the Digital World
 
-## Name of Session: 
-
-## Session Description 
+## Session Description
 
 (Why should students come to this session): Are you interested in biological enhancement? Drop in to learn about biohacking.
 
-## Session Outcomes 
-
-(What will students learn/leave with?): 
+## Session Outcomes
 
-## Session Outline 
+(What will students learn/leave with?):
 
-(Please include who is facilitating and how much time you are going to spend in each section):
+## Session Outline
 
++ Intros - 15 minutes
++ Topic 1 - Your privacy online - 15 minutes  
+ - 5 minutes presentation, 5 minute turn and talk, 5 minute group summary
+ - What do companies actually collect and what do they do with data
+   - Current laws and regulations surrounding privacy/security
 
++ Topic 2 - How to be conscious of security as you take on developer roles, how to communicate to users about privacy/security concerns
```

Comment:
Excellent:  What are the key points I need to be thinking of in security?  What are the pros/cons?  Dangers/benefits?  I love that amazon remembers everything...but do I love when they remember every search?  Do I love targeted ads?  Do I love that Alexa is probably recording me talking to my dogs every day?  (seriously I do that).